deploy/helm/sumologic/conf/logs/collector/otelcol/config.yaml

```diff
@@ -188,15 +184,10 @@ receivers:
       - type: move
         from: attributes.pod_name
         to: attributes["k8s.pod.name"]
-      - type: move
-        from: attributes.run_id
-        to: attributes["run_id"]
-      - type: move
-        from: attributes.uid
-        to: attributes["k8s.pod.uid"]
-      - type: add
-        field: attributes["fluent.tag"]
-        value: EXPR("containers." + attributes["k8s.container.name"])
```

Comment:
Just to not forget about `- filter/include_containers`

Reply:
That checks the container name:
```yaml
  filter/include_containers:
    logs:
      include:
        match_type: regexp
        record_attributes:
          - key: k8s.container.name
            value: .+
```